mxcube3/ui/reducers/sampleview.js

```diff
@@ -7,7 +7,7 @@ const initialState = {
   width: 0,
   height: 0,
   lightOn: false,
-  motors: { Light: {}, Phi: {}, PhiY: {}, PhiZ: {}, Sampx: {}, Sampy: {}, Zoom: {} },
+  motors: {},
```

Comment:
I don't agree with this change - when testing code here it didn't work at all (sample view could not be displayed) because the `motors` object was empty